packages/react-data-grid/src/common/cells/headerCells/SortableHeaderCell.tsx

```diff
@@ -52,8 +52,8 @@ export default function SortableHeaderCell<R>(props: Props<R>) {
 
   return (
     <div className="rdg-sortable-header-cell" onClick={onClick}>
-      <span className="pull-right">{SORT_TEXT[sortDirection]}</span>
       {content}
+      <span className="rdg-header-sort">{SORT_TEXT[sortDirection]}</span>
```

Comment:
Since we changed to MUI in this PR, should we change this as well to be an icon?

Reply:
Let's leave it like this for now.